2-mandatory/2-financial-times.js

```diff
@@ -6,6 +6,17 @@
 */
 function potentialHeadlines(allArticleTitles) {
```

Comment:
Looks like you have the same function name defined twice here? `function potentialHeadlines(allArticleTitles) {`